src/piggui.rs

```diff
@@ -233,13 +241,36 @@ impl Application for Gpio {
     }
 
     fn title(&self) -> String {
+        if let Some(config_filename) = &self.config_filename {
```

Comment:
You could use self.config_filename.unwrap_or("piggui") for a nice oneliner I think?